llm-server/Dockerfile

```diff
@@ -9,11 +9,11 @@ RUN npm run build
 FROM node:18-alpine as runner
 WORKDIR /my-space
 COPY --from=builder /my-space/package.json .
-COPY --from=builder /my-space/package-lock.json .
+# COPY --from=builder /my-space/package-lock.json .
```

Comment:
Hello @mrtrantuan , considering that we're utilizing 'yarn install', it's important to have a yarn.lock file present. Could you please verify that this line is adjusted to include the copying of the yarn.lock file? Your outstanding efforts are highly valued. Thank you very much.